projects/igniteui-angular/src/lib/drop-down/drop-down.component.ts

```diff
@@ -590,7 +590,8 @@ export class IgxDropDownItemNavigationDirective {
      */
     @HostListener('keydown.Space', ['$event'])
     onSpaceKeyDown(event) {
-        this.target.selectItem(this.target.focusedItem, event);
+        // V.S. : IgxDropDownComponent.selectItem needs event to be true in order to close DD as per specification
+        this.target.selectItem(this.target.focusedItem, this.target instanceof IgxDropDownComponent);
```

Comment:
Using this.target instanceof IgxDropDownComponent ? event : null can prevent problems, if in the future event inside selectItem is used and not only checked for existence